cypress/e2e/pages/topicPage/index.cy.js

```diff
@@ -1,30 +1,71 @@
-import config from '../../../support/config/services';
-import getPaths from '../../../support/helpers/getPaths';
-import serviceHasPageType from '../../../support/helpers/serviceHasPageType';
-
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
 import crossPlatformTests from './tests';
-import visitPage from '../../../support/helpers/visitPage';
-import { getTopicPagePath } from './helpers';
+import { TOPIC_PAGE } from '../../../../src/app/routes/utils/pageTypes';
+
+const topicPagesToTest = [
+  {
+    path: '/arabic/topics/cwr9j7nv58nt',
+    service: 'arabic',
+    runforEnv: ['local', 'test', 'live'],
+    pageType: TOPIC_PAGE,
+    tests: [crossPlatformTests],
+  },
+  {
+    path: '/pidgin/topics/c95y35941vrt',
+    service: 'pidgin',
+    runforEnv: ['local', 'test', 'live'],
+    pageType: TOPIC_PAGE,
+    tests: [crossPlatformTests],
+  },
+  {
+    path: '/serbian/topics/c1gd303q6y6t/lat',
+    service: 'serbian',
+    runforEnv: ['local', 'test', 'live'],
+    pageType: TOPIC_PAGE,
+    tests: [crossPlatformTests],
+    variant: 'lat',
+  },
+  {
+    path: '/ukrainian/topics/c61k92vrqz6t', // ukrainian in Russian
+    service: 'ukrainian',
+    runforEnv: ['test', 'live'],
+    pageType: TOPIC_PAGE,
+    tests: [crossPlatformTests],
+    smoke: false,
+  },
+  {
+    path: '/uzbek/topics/c8y949r98pgt/cyr',
+    service: 'uzbek',
+    runforEnv: ['local', 'test', 'live'],
+    pageType: TOPIC_PAGE,
+    tests: [crossPlatformTests],
+    variant: 'cyr',
+  },
+  {
+    path: '/uzbek/topics/c8y949r98pgt/lat',
+    service: 'uzbek',
+    runforEnv: ['local', 'test', 'live'],
+    pageType: TOPIC_PAGE,
+    tests: [crossPlatformTests],
+    variant: 'lat',
+  },
+];
 
-const pageType = 'topicPage';
-Object.keys(config)
-  .filter(service => serviceHasPageType(service, pageType))
-  .forEach(serviceId => {
-    const { variant, name: service } = config[serviceId];
+// ToDo(discuss): I put this helper here - but it seems to me that both the helper and the array of pages should be in helper.js
```

Comment:
I am hoping that we won't need this, if we remove the list from the application tests above.